release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -422,6 +428,21 @@ In the table, features are marked with the following statuses:
 AMD and Intel 64-bit architectures (x86-64-v2) will still be supported.
 ====
 
+[id="ocp-4-10-docker-config-location-deprecation"]
+==== Default Docker configuration location Deprecated
```

Comment:
Sentence case:
```suggestion
==== Default Docker configuration location deprecated
```

Or you could write "deprecation" if you wanted to match the next two entries' style 🤷